cdap-tms/src/main/java/co/cask/cdap/messaging/store/hbase/HBaseTableFactory.java

```diff
@@ -53,6 +53,7 @@
 import org.apache.hadoop.hbase.client.HBaseAdmin;
 import org.apache.hadoop.hbase.client.HTable;
 import org.apache.hadoop.hbase.client.RetriesExhaustedWithDetailsException;
+import org.apache.hadoop.hbase.regionserver.DisabledRegionSplitPolicy;
```

Comment:
It is available in all HBase versions that we supported (the class was there since very old HBase version and is publicly documented as one of the possible value of the `hbase.regionserver.region.split.policy` setting).